frontend/src/actionCreators/mineActionCreator.js

```diff
@@ -107,3 +107,19 @@ export const fetchStatusOptions = () => (dispatch) => {
       dispatch(hideLoading('modal'));
     });
 };
+
+export const fetchRegionOptions = () => (dispatch) => {
+  dispatch(request(reducerTypes.GET_REGION_OPTIONS));
+  dispatch(showLoading('modal'));
+  return axios.get(ENVIRONMENT.apiUrl + API.MINE_REGION, createRequestHeader())
```

Comment:
Thoughts on string interpolation in this codebase? I don't think i've seen any instances of it being used.

This line is likely nicer visually this way, but I'm curious about the concatenation vs interpolation pattern across the codebase.